airflow/www/templates/airflow/dags.html

```diff
@@ -88,7 +88,7 @@ <h2>DAGs</h2>
 
                 <!-- Column 4: Dag Schedule -->
                 <td>
-                    <a class="label label-default schedule {{ dag.dag_id }}" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
```

Comment:
@mik-laj Do you know if we have custom CSS somewhere based on DAG ID

Reply:
Don't think we do as dag_id can be literally anything!

Reply:
I've seen it before, but I believed it had a purpose. :-D